apps/api/src/server/build-app.js

```diff
@@ -43,7 +43,11 @@ const buildApp = (
 			return;
 		}
 
-		res.status(200).send('OK');
+		res.status(200).send({
+			status: 'OK',
+			uptime: process.uptime(),
+			commit: process.env.GIT_SHA ?? 'NO GIT SHA FOUND'
```

Comment:
We tend to pull in env vars into the config.js file so its centralised. Could we add this to there instead please? Can be added to the joi schema as optional.

Same applies to web.